infrastructure/smart-contracts/scripts/flaggedNodeLicenses.mjs

```diff
@@ -0,0 +1,31 @@
+import hardhat from "hardhat";
+import { NodeLicenseAbi, config } from "@sentry/core";
+const { ethers } = hardhat;
+
+//insert addresses to check here
+const addressesToCheck = [
+    "0x5c14ce4E3C685AFe3fC5216e390eA7f008d8A4b3"
+];
```

Comment:
We will need to input that list on runtime or we cannot use this cli comand outside of a development envirnment.

I think promting for a path to a csv on the local machine would make sense, ther is an example of this is `add-promo-code-list`

Reply:
Done! Added a prompt for the path to a csv file